lib/org/openpsa/user/handler/person/account.php

```diff
@@ -83,7 +83,11 @@ public function _handler_welcome_email(string $guid)
         $person = new midcom_db_person($guid);
         $accounthelper = new org_openpsa_user_accounthelper($person);
         $accounthelper->welcome_email();
-
+        if ($accounthelper->welcome_email()) {
+            midcom::get()->uimessages->add($this->_l10n->get('org.openpsa.user'), sprintf($this->_l10n->get('password reset and mail to %s sent'), $person->email ));
+        } else {
+            midcom::get()->uimessages->add($this->_l10n->get('org.openpsa.user'), sprintf($this->_l10n->get('error password reset and mail not sent to %s'), $person->email) , 'warning'); 
```

Comment:
btw, usually we write `$this->_component` instead of the literal component name. That makes it easier to extend